concordia/static/js/action-app/index.js

```diff
@@ -442,6 +442,26 @@ export class ActionApp {
                 );
                 this.updateAvailableCampaignFilters();
             });
+        fetchJSON(this.config.urls.topicList)
+            .then(data => {
+                data.objects.forEach(topic => {
+                    let o = document.createElement('option');
+                    o.value = topic.id;
+                    o.textContent = topic.title;
+
+                    // TODO: this does not handle the case where the last assets of a campaign change state while the app is open
+                    Object.entries(topic.asset_stats).forEach(
+                        ([key, value]) => {
+                            o.dataset[key] = value;
+                        }
+                    );
+
+                    this.campaignSelect.appendChild(o);
```

Comment:
I would prefer not to lump these into the campaign selector since it seems likely to confuse both developers and users but if we're doing that I'd think that'd be something like updating the API data to include the topic, tagging these options to indicate whether they're a campaign or a topic (and since this cost is almost entirely identical to the previous block we probably want to move it into a shared function which could be called with a class attribute for the `option` element), modifying `getVisibleAssets()` to filter topics as well as campaigns, and possibly having `updateAvailableCampaignFilters()` sort the list.